webpack/scenes/RedHatRepositories/components/EnabledRepository/EnabledRepositoryContent.js

```diff
@@ -4,7 +4,9 @@ import cx from 'classnames';
 import { Spinner, OverlayTrigger, Tooltip } from 'patternfly-react';
 import { translate as __ } from 'foremanReact/common/I18n';
 
-const EnabledRepositoryContent = ({ loading, disableTooltipId, disableRepository }) => (
+const EnabledRepositoryContent = ({
+  loading, disableTooltipId, disableRepository, canDisable,
+}) => (
   <Spinner loading={loading} inline>
     <OverlayTrigger
       overlay={<Tooltip id={disableTooltipId}>{__('Disable')}</Tooltip>}
```

Comment:
I think it would be nice to state the reason for disablement here. It took me too long to find the help text at the top.

```suggestion
      overlay={<Tooltip id={disableTooltipId}>{canDisable ? __('Disable') : __('Cannot be disabled because it is part of a published content view')}</Tooltip>}
```

Reply:
I had tried this..The tooltip doesn't always show up on a disabled button for some reason..

Reply:
Pushed an update..The tooltip isn't very reliable..Maybe my chrome..Not sure..

Reply:
Seems to work fine for me on Chrome! :+1: